chatmeleon-server/tsconfig.json

```diff
@@ -1,16 +1,14 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "esnext",
+    "module": "commonjs",
     "esModuleInterop": true, //This is required since some of the dependencies are CommonJS packages.
-    "moduleResolution": "NodeNext", // required,
+    "moduleResolution": "node", // required,
     "outDir": "./dist",
-    "rootDir": "./src",
+    "rootDir": "./",
     "strict": true
   },
-  "ts-node": {
-    "esm": true,
-    "experimentalSpecifierResolution": "node"
-  },
-  "include": ["src"]
-}
+  "include": [
+    "./"
+  ]
```

Comment:
Change this back to `"./src"` after moving all code files into `src`

Reply:
I got it. But can you explain why we need that as currently there is no problem running the program.

Reply:
https://www.typescriptlang.org/tsconfig#include

Actually, we can remove this as it doesn't add anything with our current set up.

Reply:
@XkhangNguyen I guess just remove `include` cuz it's redundant in this case